app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -308,6 +307,19 @@ def self.evacuate_queue(userid, vm, options = {})
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
 
+  def resize_queue(userid, options = {})
+    options[:src_ids] = [id]
+    # TODO: @requestId - figure out what to do. old code in resize:
+    #   if params[:id] && params[:id] != 'new'
```

Comment:
@agrare is it possible to edit a request before it's completed? Trying to understand if we need to handle the case where the request already exists

Reply:
Earlier today I was trying to figure out what the request was buying us, typically these are a simple put the message on the queue and execute it in the right zone.  There might be some approval workflow logic that you could hook into it

I don't believe you can edit a request before it is completed.

Reply:
I've thought about this too. If `vm_resize` is not handled as a request, then wouldn't that give any user with access to the feature the ability to scale up without limit? If so maybe the request overhead is worth it.